SearchImplem/City.swift

```diff
@@ -10,13 +10,10 @@ import Foundation
 
 class City: NSObject {
     
-    let name: String
-    let country: String
-    let adminName: String
+    var name: String?
+    var country: String?
+    var adminArea: String?
+    var coordinates: (Double, Double)?
+    var googleLocationID: String?
```

Comment:
You should have a subclass of City (GoogleCity?) that defines this googleLocationID. it is only used for the Google API

Reply:
Done